Vagrantfile

```diff
@@ -57,11 +63,42 @@
 #
 # Customization Options
 # =====================
-ALAVETELI_FQDN = ENV['ALAVETELI_VAGRANT_FQDN'] || "alaveteli.10.10.10.30.xip.io"
-ALAVETELI_MEMORY = ENV['ALAVETELI_VAGRANT_MEMORY'] || 1536
-ALAVETELI_THEMES_DIR = ENV['ALAVETELI_THEMES_DIR'] || '../alaveteli-themes'
-ALAVETELI_OS = ENV['ALAVETELI_VAGRANT_OS'] || 'wheezy64'
-ALAVETELI_USE_NFS = ENV['ALAVETELI_VAGRANT_USE_NFS'] || false
+#
+# Defaults can be overridden either in `.vagrant.yml` with the same key name, or
+# via the environment by prefixing the key with `ALAVETELI_VAGRANT_` and
+# upcasing. Boolean values can be set to `false` in the environment with "0",
+# "false" or "no". The environment takes precedence over `.vagrant.yml`.
+DEFAULTS = {
+  'fqdn' => 'alaveteli.10.10.10.30.xip.io',
```

Comment:
Note that I used `String` keys here because that's what standard YAML keys (e.g. `fqdn:` get parsed as). To get `Symbol` keys you need to use `:fqdn:` which is not obvious at all, so we'll just stick with strings.